### ISSUE TITLE
Question: What are those lines used for?

### ISSUE THREAD

**Issue by Bozenton** [bit-bots/humanoid_base_footprint#42]
Hi there, 
Thanks for your sharing of the repo, I am reading your code and notice those lines: 
https://github.com/ros-sports/humanoid_base_footprint/blob/2e9c85bc55bb8bcf4bc3f121e1f645ce2325ffd7/src/base_footprint.cpp#L140-L152
It seems that those lines are never used later in the other part of the code. So I am wondering what they are used for?

Hope for your answer sincerely.

**Comment by SammyRamone** [comment]
As far as I can see it is not used. It looks like this was first coded to get the orientation but was then solved by another approach in the lines below. I guess it can just be removed. Maybe @jgueldenstein can have a look to make sure

**Comment by jgueldenstein** [comment]
I have not touched this code in a very long time. I think @Flova would be the best candidate to know why this is there. It also seems to me that it can be deleted as it is not used afterwards.

**Comment by Flova** [comment]
Good catch. There is some duplication there. I would prefer using the rot_conv based version that is currently unused. Both methods just remove the yaw rotation component from the odometry, to estimate the pitch and roll of the robot relative to the base footprint frame on the ground.
